taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -91,10 +91,8 @@ def add_nightly_version(config, tasks):
 def add_release_version(config, tasks):
     for task in tasks:
         if task.pop("include-release-version", False):
-            # TODO Move GIT_TAG to a parameter
-            git_tag = os.environ['GIT_TAG']
-            version = git_tag[1:]  # remove prefixed "v"
-
+            tag = config.params["head_tag"]
+            version = tag[1:]  # remove prefixed "v"
```

Comment:
I would put the version in the parameters, and just use that here.

Reply:
I wasn't sure about the duplication of data, but I now agree with you. Done!